hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFutureTest.java

```diff
@@ -36,91 +36,52 @@ public void setup() {
         operationService = getOperationService(local);
     }
 
-    // ================= isDone ==========================================
-
     @Test
     public void isDone_whenNullResponse() throws ExecutionException, InterruptedException {
         DummyOperation op = new DummyOperation(null);
 
         InternalCompletableFuture future = operationService.invokeOnTarget(null, op, getAddress(local));
-        // first we wait for the future to complete.
-        future.get();
+        future.get(); // first we wait for the future to complete.
```

Comment:
Perhaps we should update checkstyle. I have little problems with a trailing comment. Can you suggest it on the dev channel?